src/app/legacy/containers/SocialEmbed/index.stories.jsx

```diff
@@ -22,7 +22,7 @@ import withContexts from './common/testHelper';
 
 // eslint-disable-next-line react/prop-types
 const BackgroundColorWrapper = ({ children }) => (
-  <div css={css({ backgroundColor: C_GREY_2, padding: 20 })}>{children}</div>
+  <div css={css({ backgroundColor: GREY_2, padding: 20 })}>{children}</div>
```

Comment:
Direct import used. Not sure the best way to access the theme here.

Reply:
I'd say it's fine as it's a story